generator/internal/genclient/parser/openapi/openapi.go

```diff
@@ -247,6 +248,7 @@ func makeMethods(api *genclient.API, model *libopenapi.DocumentModel[v3.Document
 				OutputTypeID:  responseMessage.ID,
 				PathInfo:      pathInfo,
 			}
+			api.State.MethodByID[m.ID] = m
```

Comment:
`m.ID` does not have the same format as the Protobuf methods. Those include the service ID too.  Probably does not matter (maybe for documentation links? not sure).

Reply:
made them align, thanks